### ISSUE TITLE
bin repr of x,y coordinate opposite of spaxel repr x,y

### ISSUE THREAD

**Issue by havok2063** [sdss/marvin#457]
**Describe the bug**
The repr in the `Bin` object has the opposite x, y coordinates compared to the `Spaxel` repr.  At least in the `HYB10` binning for both MPL-7 and 6.  `SPX` binning in MPL-6 seems ok.  Might be underlying data and not just repr.  The spaxels loaded in the Bin are the opposite of what I get when loading an unbinned `Spaxel`    

**To Reproduce**
```
cube = Cube('8485-1901')

# HYB10 binning
maps = cube.getMaps(bintype='HYB10')
maps
<Marvin Maps (plateifu='8485-1901', mode='local', data_origin='db', bintype='HYB10', template='GAU-MILESHC')>

# get the Bin at 22, 14
s = maps[22,14]
<Marvin Bin (plateifu=8485-1901, x=14, y=22; x_cen=-3, y_cen=5, n_spaxels=4, loaded=cube/maps)>

# Look at the spaxels in that Bin
s.spaxels
[<Marvin Spaxel (x=21, y=13, loaded=False),
 <Marvin Spaxel (x=21, y=14, loaded=False),
 <Marvin Spaxel (x=22, y=13, loaded=False),
 <Marvin Spaxel (x=22, y=14, loaded=False)]

# SPX binning
m = cube.getMaps()
m
<Marvin Maps (plateifu='8485-1901', mode='local', data_origin='db', bintype='SPX', template='GAU-MILESHC')>
m[22,14]
<Marvin Spaxel (plateifu=8485-1901, x=14, y=22; x_cen=-3, y_cen=5, loaded=cube/maps)>

```
**Expected behavior**
The same x, y, representation and/or spaxel data.

**Additional context**
Marvin tested in `dr15` branch and in tag `2.2.5`.

**Comment by albireox** [comment]
Yep, this is a bug. The problem is in `Bin._create_spaxels`

```python
        spaxel_coords = zip(*np.where(binid_map == self.binid))
        self.spaxels = []

        for jj, ii in spaxel_coords:
            self.spaxels.append(Spaxel(x=jj, y=ii, plateifu=self.plateifu, release=self.release,
                                       cube=self._cube, maps=True if self._maps else False,
                                       modelcube=True if self._modelcube else False,
                                       bintype=None, template=self.template, lazy=True))
```

I'm inverting the output twice once in the for and another when calling `Spaxel`. It should be `for iim jj in spaxel_coords` and the rest can stay as it is. So, f*ck.

I'll add some tests and fix this ...

**Comment by albireox** [comment]
Fixed in #458
